server/src/test/java/io/druid/server/audit/SQLAuditManagerTest.java

```diff
@@ -135,6 +134,90 @@ public void testFetchAuditHistory() throws IOException
     Assert.assertEquals(entry, auditEntries.get(1));
   }
 
+  @Test
```

Comment:
Can you add tests for the count <=0 case? The docs do not clarify what happens in such a case and a unit test would be helpful.